src/DynamicNLPModels.jl

```diff
@@ -203,53 +205,30 @@ struct SparseLQDynamicModel{T, V, M1, M2, M3, MK} <:  AbstractDynamicModel{T,V}
   dynamic_data::LQDynamicData{T, V, M3, MK}
 end
 
-struct DenseLQDynamicBlocks{T, M1, M2}
-    A::M1
-    B::M1
-    Q::M2
-    R::M2
-    S::M2
-    K::M2
-    E::M1
-    F::M1
-    gl::M1
-    gu::M1
+struct DenseLQDynamicBlocks{T, M}
+    A::M
+    B::M
+
 end
```

Comment:
Maybe good to add a docstring of explaining what are these. Or we can even point to the reference